### PR TITLE
improve welcome message

### PR DESCRIPTION
**Pull request by mvo5**
Trivial branch to help new classic users understand how it works.

### CODE REVIEW

**Comment by zyga** [comment]
+1

File: cmd/snappy/cmd_shell.go [hunk 1]
```diff
@@ -73,6 +73,12 @@ Use "sudo snappy enable-classic" to enable it.`))
 		}
 
 		fmt.Println(i18n.G(`Entering classic dimension`))
+		fmt.Println(i18n.G(`
+
+The home directory is shared between snappy and the classic dimension.
+Run the "exit" command to leave the classic shell again. All files
+created will be preserved.
```
Comment (niemeyer) [inline cmd/snappy/cmd_shell.go]: I'd suggest this shorter version:
```
The home directory is shared between snappy and the classic dimension.
Run "exit" to leave the classic shell.
```
The rationale being that data preservation is a touchy topic. If we say it's preserved, I'd be tempted to expand with caveats, which are boring an most likely uninteresting for the typical user.

**Comment by niemeyer** [comment]
LGTM, with a minor tweak suggestion above.

**Comment by elopio** [comment]
Please update the copyright of cmd/snappy/cmd_shell.go to include 2016.

**Comment by mvo5** [comment]
Thanks for the reivew! I addressed the points (including the copyright notice).

**Comment by elopio** [comment]
This has like a +3, so I'll land.

**Pull request merged**